app/src/main/java/rick_and_morty/ui/characters/composables/CharacterList.kt

```diff
@@ -0,0 +1,84 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import rick_and_morty.ui.characters.CharactersViewModel
+
+@Composable
+fun CharacterList() {
+    val charactersViewModel = viewModel(modelClass = CharactersViewModel::class.java)
+
+    var page = 1;
+
+    charactersViewModel.getCharacters(page)
+
+    val characters = charactersViewModel.characters.collectAsState(initial = emptyList())
+
+    LazyColumn (
+        verticalArrangement = Arrangement.spacedBy(5.dp)
+    ) {
+
+        items(characters.value) {characters ->
+            CharacterRow(characterResultsDto = characters)
+        }
+        item {
+            LaunchedEffect(true) {
+                if (!charactersViewModel.initialLoad) {
+                    page += 1
+                    charactersViewModel.getCharacters(page)
```

Comment:
Do you think you're going to implement pagination here? If so, I guess you will need to implement a reducer here or IMO in the ViewModel

Reply:
For now there is no pagination, and it will be just scrolling down, and new list of characters will pop up